leader-election-api/src/main/java/com/palantir/leader/LeaderElectionService.java

```diff
@@ -51,13 +53,15 @@
      *
      * @return a leadership token to be used with {@link #isStillLeading}
      */
+    @Timed
```

Comment:
don't think this is interesting, as it includes time where you'd never become the leader, so metric is itself never useful.

Reply:
yeah, we probably want something more targetted towards "how long until we elected a new leader"

Reply:
good call, will remove